store/cachekv/store.go

```diff
@@ -339,7 +339,7 @@ func (store *Store) dirtyItems(start, end []byte) {
 		}
 	}
 
-	kvL := make([]*kv.Pair, 0)
+	kvL := make([]*kv.Pair, 0, endIndex-startIndex)
```

Comment:
```suggestion
	kvL := make([]*kv.Pair, 0, endIndex-startIndex+1)
```
this is more precise I think.

Reply:
Thanks @yihuang, I made the update!